lib/ocpp/v201/charge_point.cpp

```diff
@@ -3818,6 +3849,36 @@ void ChargePoint::execute_change_availability_request(ChangeAvailabilityRequest
     }
 }
 
+// K01.27 - load profiles from database
+void ChargePoint::load_charging_profiles() {
+    try {
+        auto evses = this->database_handler->get_all_charging_profiles_by_evse();
+        EVLOG_info << "Found " << evses.size() << " evse in the database";
+        for (auto& profiles : evses) {
```

Comment:
You can use `for (auto& [evse_id, profile] : evses)` here for increased readability.